src/app/programs/projects.service.ts

```diff
@@ -0,0 +1,66 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { parseString } from 'xml2js';
+
+export class ProgramItem {
+  title: string;
+  link: string;
+  description: string;
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProjectsService {
+  public smallBasicFeedUrl = "https://techcommunity.microsoft.com/gxcuf89792/rss/board?board.id=SmallBasic";
+  private corsUrl = "https://cors-anywhere.herokuapp.com/";
+  private smallBasicFeed: Promise<ProgramItem[]>;
+
+  constructor(private http: HttpClient) { }
+
+  public getPrograms(): Promise<ProgramItem[]> {
```

Comment:
If this is the same code as the service that fetches items for the home page, don't duplicate it. Move it into a src/app/commonservices folder and use the same service for both components